spec/providers/database.spec.ts

```diff
@@ -229,6 +229,32 @@ describe('DeltaSnapshot', () => {
       subject.forEach(counter);
       expect(count).to.eq(0);
     });
```

Comment:
Can you add some checks here that the return values of both `forEach()` calls in this test are `false`?